Makefile

```diff
@@ -257,31 +358,117 @@ changelog-unreleased: ## show the current changelog that will be produced on the
 			-t 748.5989 \
 			/CHANGELOG.md
 
-.PHONY: release
-release: clean-dist ci-bootstrap-mac changelog-release ## Build and publish final binaries and packages. Intended to be run only on macOS.
-	$(call title,Publishing release artifacts)
-
-	# Prepare for macOS-specific signing process
-	.github/scripts/mac-prepare-for-signing.sh
-
-	# create a config with the dist dir overridden
-	echo "dist: $(DISTDIR)" > $(TEMPDIR)/goreleaser.yaml
-	cat .goreleaser.yaml >> $(TEMPDIR)/goreleaser.yaml
-
-	# release (note the version transformation from v0.7.0 --> 0.7.0)
-	bash -c "\
-		BUILD_GIT_TREE_STATE=$(GITTREESTATE) \
-		VERSION=$(VERSION:v%=%) \
-		$(TEMPDIR)/goreleaser \
-			--rm-dist \
-			--config $(TEMPDIR)/goreleaser.yaml \
-			--release-notes <(cat CHANGELOG.md)"
-
-	# verify checksum signatures
-	.github/scripts/verify-signature.sh "$(DISTDIR)"
+.PHONY: homebrew-formula-generate
+.SILENT: homebrew-formula-generate
+homebrew-formula-generate:
+	$(call title,Generating homebrew formula)
+	# dependencies: curl, jq, openssl
+
+	RELEASE_URL="https://api.github.com/repos/anchore/$(BIN)/releases/tags/$(VERSION_TAG)" && \
+	echo "Using release: $${RELEASE_URL}" && \
+	curl -sSL "$${RELEASE_URL}" > "$(TEMPDIR)/release.json"
+
+	export DARWIN_AMD64_ASSET_URL=$(call asset_url,"$(TEMPDIR)/release.json","darwin_amd64.zip") && \
+	curl -sSL "$${DARWIN_AMD64_ASSET_URL}" > "$(TEMPDIR)/darwin_amd64_asset" && \
+	export DARWIN_AMD64_ASSET_SHA256=$(call sha256,"$(TEMPDIR)/darwin_amd64_asset") && \
+	\
+	export LINUX_AMD64_ASSET_URL=$(call asset_url,"$(TEMPDIR)/release.json","linux_amd64.tar.gz") && \
+	curl -sSL "$${LINUX_AMD64_ASSET_URL}" > "$(TEMPDIR)/linux_amd64_asset" && \
+	export LINUX_AMD64_ASSET_SHA256=$(call sha256,"$(TEMPDIR)/linux_amd64_asset") && \
+	\
+	export VERSION=$(call get_version_from_version_tag,$(VERSION_TAG)) && \
+	\
+	cat "./.homebrew-formula-template.rb" | \
+		envsubst > "$(HOMEBREW_FORMULA_FILE)"
```

Comment:
`envsubst` is not available everywhere I think, and some systems require you to install the `gettext` package(s) to make it available. Is this dependency accounted for somewhere?

Reply:
This is a great point, it's not accounted for or explicitly checked for — I'll change that

Reply:
Replaced with a templated heredoc instead